frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/ticketView/ticketView.component.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Viewer as ViewerService } from '@/v4/services/viewer/viewer';
+import EditViewpointIcon from '@assets/icons/outlined/rotate_arrow-outlined.svg';
+import CreateViewpointIcon from '@assets/icons/outlined/add_circle-outlined.svg';
+import GotoViewpointIcon from '@assets/icons/outlined/aim-outlined.svg';
+
+import { stripBase64Prefix } from '@controls/fileUploader/imageFile.helper';
+import { MenuItem } from '@mui/material';
+import { useEffect } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { isEmpty, isEqual } from 'lodash';
+import { getImgSrc } from '@/v5/store/tickets/tickets.helpers';
+import { ActionMenuItem } from '@controls/actionMenu';
+import { Viewpoint } from '@/v5/store/tickets/tickets.types';
+import { BasicTicketImage } from '../basicTicketImage/basicTicketImage.component';
+import { TicketImageAction } from '../basicTicketImage/ticketImageAction/ticketImageAction.component';
+import { ActionMenu } from '../basicTicketImage/ticketImageAction/ticketImageAction.styles';
+import { TicketImageActionMenu } from '../basicTicketImage/ticketImageActionMenu.component';
+
+type ITicketView = {
+	value: Viewpoint | undefined;
+	label: string;
+	error: boolean;
+	helperText: string;
+	required: boolean;
+	onBlur: () => void;
+	onChange: (newValue) => void;
+};
+
+export const TicketView = ({
+	value,
+	error,
+	helperText,
+	required,
+	onBlur,
+	onChange,
+	...props
+}: ITicketView) => {
+	const updateViewpoint = async () => {
+		const currentViewpoint = await ViewerService.getViewpoint();
+		if (isEqual(currentViewpoint, value)) return;
+		onChange?.(currentViewpoint);
+	};
+
+	const goToViewpoint = async () => {
+		if (!value) return;
```

Comment:
This method is only used by the respective ticketAction, which is disabled when `!(value?.camera)`. Could we perhaps change the disabled condition in the action to be `disabled={!value}` and get rid of this `if`?

Reply:
There is already that condition to disable the goto button . So this line is not necessary.